tests/tests_unit/rules/test_analysis.py

```diff
@@ -1,32 +1,40 @@
-from cognite.neat.rules._analysis import (
-    get_class_linkage,
-    get_connected_classes,
-    get_defined_classes,
-    get_disconnected_classes,
-    get_symmetric_pairs,
-)
+from cognite.neat.rules._analysis._information_rules import InformationArchitectRulesAnalysis
 from cognite.neat.rules.models._rules import InformationRules
 from cognite.neat.rules.models._rules._types import ClassEntity
 
 
 class TestRulesAnalysis:
     def test_defined_classes(self, david_rules: InformationRules) -> None:
```

Comment:
Should probably add a few more tests here of the public methods in the `InformationArchitectRulesAnalysis`.

Reply:
done!